root/defaults/nginx/server-confs/ssl.conf.sample

```diff
@@ -1,40 +1,41 @@
-## Version 2021/09/19 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/ssl.conf
+## Version 2021/11/25 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/server-confs/ssl.conf.sample
 
 ### Mozilla Recommendations
-# generated 2020-06-17, Mozilla Guideline v5.4, nginx 1.18.0-r0, OpenSSL 1.1.1g-r0, intermediate configuration
-# https://ssl-config.mozilla.org/#server=nginx&version=1.18.0-r0&config=intermediate&openssl=1.1.1g-r0&guideline=5.4
+# generated 2021-10-16, Mozilla Guideline v5.6, nginx 1.20.1-r3, OpenSSL 1.1.1l-r0, intermediate configuration
+# https://ssl-config.mozilla.org/#server=nginx&version=1.20.1-r3&config=intermediate&openssl=1.1.1l-r0&guideline=5.6
 
+listen 443 ssl http2;
+listen [::]:443 ssl http2;
+
+ssl_certificate /config/keys/letsencrypt/fullchain.pem;
+ssl_certificate_key /config/keys/letsencrypt/privkey.pem;
 ssl_session_timeout 1d;
-ssl_session_cache shared:MozSSL:10m;  # about 40000 sessions
+ssl_session_cache shared:MozSSL:10m; # about 40000 sessions
 ssl_session_tickets off;
 
+# curl https://ssl-config.mozilla.org/ffdhe2048.txt > /path/to/dhparam
+ssl_dhparam /config/nginx/dhparams.pem;
```

Comment:
Alternatively to the last comment, this should be commented out if we don't ship dhparams by default

Reply:
Still shipping via https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/80618475ed4d591f938db2e9b861c72274b0819d/root/etc/cont-init.d/13-nginx#L29-L31 in the base. The ssl.conf in swag is not identical to the ssl.conf in the base because of the cert paths. I'm considering using variable replacement or sed in the file to allow us to maintain the ssl.conf in the base and remove it from swag.